contentcuration/contentcuration/utils/assessment/base.py

```
@@ -0,0 +1,382 @@
+    def handle_after_assessment_items(self):
+        pass
+
+    def _create_zipfile(self):
+        filename = "{0}.{ext}".format(self.ccnode.title, ext=self.file_format)
+        with NamedTemporaryFile(suffix="zip") as tempf:
+            with zipfile.ZipFile(tempf.name, "w") as zf:
+                for file_path in self.files_to_write:
+                    self.write_to_zipfile(
+                        zf,
+                        os.path.relpath(file_path, self.tempdir),
+                        open(file_path, "rb").read(),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)